src/serac/physics/solid_functional.hpp

```diff
@@ -368,17 +367,16 @@ class SolidFunctional<order, dim, Parameters<parameter_space...>, std::integer_s
             deformation_grad = deformation_grad + du_dX;
           }
 
-          // Note that the jacobian needs the fixup for the shape displacement.
-          // The material returns Kirchoff stress, which contains det(du / dX'). We want
-          // The final Jacobian to be det(du / dX), so we compute
-          // det(du / dX) = det(du / dX') * det(dX' / dX) = det(du / dX') * det(I + dp / dX)
-          auto flux = dot(stress, transpose(inv(deformation_grad))) * det(I + dp_dX);
+          auto flux = dot(stress, transpose(inv(deformation_grad))) * det(deformation_grad);
 
```

Comment:
For future reference, if we're shifting to Cauchy stress, we should think about using an updated Lagrangian approach, computing the shape function derivatives w.r.t. current coordinates. Then we would cleanly return Cauchy stress from here. (We still have the shape displacement stuff to deal with until we move that into the integration as well).